### PR TITLE
feat: get usbmuxd socket address from USBMUXD_SOCKET_ADDRESS environment

### PR DESCRIPTION
**Pull request by harlentan**
follow [libusbmuxd](https://github.com/libimobiledevice/libusbmuxd), get usbmuxd socket address from USBMUXD_SOCKET_ADDRESS environment

### CODE REVIEW

File: lib/usbmux/index.js [hunk 1]
```diff
@@ -62,13 +63,25 @@ function swap16 (val) {
...
 async function getDefaultSocket (opts = {}) {
-  const {
+  let {
     socketPath = DEFAULT_USBMUXD_SOCKET,
     socketPort = DEFAULT_USBMUXD_PORT,
     socketHost = DEFAULT_USBMUXD_HOST,
     timeout = 5000,
   } = opts;
 
+  let usbmuxdSocketAddress = process.env.USBMUXD_SOCKET_ADDRESS;
```
Comment (KazuCocoa) [inline lib/usbmux/index.js:73]: `const`?

File: README.md [hunk 2]
```diff
@@ -33,6 +33,9 @@ More information can be found at the links below:
 - `services.startNotificationProxyService`
 - `services.startHouseArrestService`
 
+### Environment
+- `USBMUXD_SOCKET_ADDRESS` follow [libusbmuxd](https://github.com/libimobiledevice/libusbmuxd) to get usbmuxd socket address from USBMUXD_SOCKET_ADDRESS environment.
```
Comment (mykola-mokhnach) [inline README.md:37]: Please describe how an example/acceptable value should look like and a purpose of it

File: lib/usbmux/index.js [hunk 3]
```diff
@@ -62,13 +63,25 @@ function swap16 (val) {
...
+  if (usbmuxdSocketAddress) {
+    let parsedAddr = url.parse(usbmuxdSocketAddress);
+    if (parsedAddr.protocol === 'unix:') {
+      socketPath = parsedAddr.pathname;
+    } else {
+      let ipPortPair = usbmuxdSocketAddress.split(':');
```
Comment (mykola-mokhnach) [inline lib/usbmux/index.js:79]: should we validate first if `usbmuxdSocketAddress` string is formatted properly?

File: lib/usbmux/index.js [hunk 4]
```diff
@@ -62,13 +63,25 @@ function swap16 (val) {
...
+      socketHost = ipPortPair[0];
```
Comment (mykola-mokhnach) [inline lib/usbmux/index.js:80]: this could be simplified to 
```
[socketHost, socketPort] = usbmuxdSocketAddress.split(':');
```

**Comment by mykola-mokhnach** [comment]
Closed because of no response

**Pull request closed without merging**